data/src/main/java/com/microsoft/azure/kusto/data/ClientImpl.java

```diff
@@ -22,8 +22,8 @@
     private static final String QUERY_ENDPOINT_VERSION = "v2";
     private static final String STREAMING_VERSION = "v1";
     private static final String DEFAULT_DATABASE_NAME = "NetDefaultDb";
-    private static final Long COMMAND_TIMEOUT_IN_MILLISECS = TimeUnit.MINUTES.toMillis(10) + TimeUnit.SECONDS.toMillis(30);
-    private static final Long QUERY_TIMEOUT_IN_MILLISECS = TimeUnit.MINUTES.toMillis(4) + TimeUnit.SECONDS.toMillis(30);
+    private static final Long COMMAND_TIMEOUT_IN_MILLISECS = TimeUnit.MINUTES.toMillis(10);
```

Comment:
Because we now add CLIENT_SERVER_DELTA_IN_MILLISECS to all timeouts.
Admittedly this doesn't belong in this PR, but I thought it was small enough to not be worth the extra overhead :-)
WDYT?

Reply:
sounds ok